source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing.md

```diff
@@ -48,13 +48,38 @@ Insert into **Jormun** configuration:
             "service_url": "https://via.klaxit.com/v1/carpoolJourneys",
             "api_key": "Token",
             "network": "Klaxit",
-            "timeout": 10,            # optional
-            "timedelta": 3600,        # optional
+            "timeout": 10,              # optional (defaults to 2 in secs)
+            "timedelta": 3600,          # optional (defaults to 3600 in secs)
+            "departure_radius": 3,      # optional (defaults to 2 in km)
+		    "arrival_radius": 3         # optional (defaults to 2 in km)
```

Comment:
I don't really understand the comments "defaults to" because they don't match the values in the text"

Reply:
Since the parameter 'departure_radius' is optional, it may be absent and in that case we use 2 km as default value as in https://github.com/CanalTP/navitia/blob/72d8f44b49aeddaefb016f5855a1623f2d7ffa35/source/jormungandr/jormungandr/scenarios/ridesharing/karos.py#L63

Reply:
Ok I get it :+1: 
Maybe just remove the "in"s : 
```suggestion
            "timeout": 10,              # optional (defaults to 2  secs)
            "timedelta": 3600,          # optional (defaults to 3600  secs)
            "departure_radius": 3,      # optional (defaults to 2 km)
	    "arrival_radius": 3         # optional (defaults to 2 km)
```